.github/workflows/e2e.yaml

```diff
@@ -12,6 +12,10 @@ jobs:
         uses: actions/checkout@v3
       - name: compile clusterkubevirtadm
         run: make clusterkubevirtadm-linux
+      - name: Install Go
```

Comment:
does `make clusterkuevirtadm-linux` need a compiler? If so, the order probably has to be changed.

Reply:
sure. fixing